src/Materials/Textures/rawCubeTexture.ts

```diff
@@ -81,6 +81,15 @@ export class RawCubeTexture extends CubeTexture {
 
     /** @hidden */
     public static _UpdateRGBDAsync(internalTexture: InternalTexture, data: ArrayBufferView[][], sphericalPolynomial: Nullable<SphericalPolynomial>, lodScale: number, lodOffset: number): Promise<void> {
+        internalTexture.onRebuildCallback = (_internalTexture) => {
+            const proxy = internalTexture.getEngine().createRawCubeTexture(null, internalTexture.width, internalTexture.format, internalTexture.type,
+                internalTexture.generateMipMaps, internalTexture.invertY, internalTexture.samplingMode, internalTexture._compression);
+            return {
+                proxy: EnvironmentTextureTools._UpdateRGBDAsync(proxy, internalTexture._bufferViewArrayArray!, internalTexture._sphericalPolynomial, internalTexture._lodGenerationScale, internalTexture._lodGenerationOffset),
+                isReady: true,
+                isAsync: true
+            };
+        };

```

Comment:
Maybe this function (the code below) can be implemented by simply calling `EnvironmentTextureTools._UpdateRGBDAsync(...)`, as it is doing exactly the same thing?

Reply:
yes, if this is merged I will do the cleanup work in the other draft PR that organizes the public-static classes. Otherwise I will have many conflicts

Reply:
strike that. I changed the function :-)
Thanks!